lib/Dialect/AIEVec/Transforms/VectorToVectorConversions.cpp

```diff
@@ -411,6 +460,107 @@ struct FlattenMultDimTransferWritePattern
   }
 };
 
+// This pattern takes out an implicit transposition of the `rhs` operand in a
```

Comment:
I suggest to clarify the comment as follows:
 an implicit transposition of the `rhs` operand
=>
 an implicit transposition of the 2 innermost dimensions of the `rhs` operand